src/cromshell/utilities/cromshellconfig.py

```diff
@@ -151,9 +150,14 @@ def __load_cromshell_config_file(config_directory, config_file_name):
 
     cromshell_config_path = os.path.join(config_directory, config_file_name)
     if not Path(cromshell_config_path).exists():
-        LOGGER.error("Cromshell config file %s was not found", cromshell_config_path)
-        LOGGER.error("Please create %s", cromshell_config_path)
-        raise Exception(f"Cromshell config file {cromshell_config_path} was not found")
+        LOGGER.info("Cromshell config file %s was not found", cromshell_config_path)
+        LOGGER.info("Creating %s", cromshell_config_path)
+
+        config_template = '{\n"cromwell_server": "[enter server url]"\n}'
```

Comment:
If we have to generate a configuration file anyway, we should write out all the options with default values so the user knows what options exist by looking at the config file.

Reply:
@jonn-smith  Done
This [variable](https://github.com/broadinstitute/cromshell/pull/156/files#diff-82107d55dd3286d108a0a9d136339c3056207f177a7d9c3d23daf3433b1c17e1R36) will hold all possible options and updated this [line](https://github.com/broadinstitute/cromshell/pull/156/files#diff-82107d55dd3286d108a0a9d136339c3056207f177a7d9c3d23daf3433b1c17e1R163) accordingly shift it from dict to json